packages/floating-label/README.md

```diff
@@ -0,0 +1,68 @@
+# React Floating Label
+
+MDC React Floating Label is as a React container around MDC Floating Label. Please see [MDC Fab](https://github.com/material-components/material-components-web/tree/master/packages/mdc-floating-label).
+
+## Installation
+
+```
+npm install @material/react-floating-label
+```
+
+## Props
+
+Prop Name | Type | Description
+--- | --- | ---
+className | String | Classes to be applied to the root element.
+onShakeEnd | Function | Callback method called when the shake animation ends.
+setWidth | Function | Callback method to pass the width to a parent Component.
+shouldFloat | Boolean | Floats label depending on value passed.
+shouldShake | Boolean | Shakes label depending on value passed.
+
+## Usage
+
+### Positioning
+
+React Floating Label is meant to be used with other Components, specifically input type elements. Floating label is positioned absolute, which requires the parent element to be position _relative_ or _absolute_.
```

Comment:
Just say
which requires the parent element to be position _relative_

You don't have to explain that position: absolute makes the elements children